_config.yml

```diff
@@ -71,6 +71,17 @@ primary_navigation:
     url:
       en: "/resources/"
       es: "/resources/"
+    children:
+    - name:
+        en: Guidance & Resource Materials
+        en-short: Resources
+        es: Guidance & Resource Materials
+      url: "/resources/"
+    - name:
+        en: Help
+        en-short: Help
+        es: Help
+      url: "/help/"
```

Comment:
Just checking: are `en` and `es` here supposed to be the same? Is there some other system that is handling the translation for us?

Reply:
This is where we'd put the translations, but that would be separate work to procure those - even for a single word like this - so we'll default to en for now.